pages/documentation/developer/repositories/powsybl-single-line-diagram.md

```diff
@@ -6,11 +6,13 @@ This [repository](https://github.com/powsybl/powsybl-single-line-diagram) provid
 
 ## Features
 
-- creation of single line diagrams for given voltage levels
+- creation of single line diagrams for given voltage levels or substations
```

Comment:
I think there are more features in that repository:
- a small demo app,
- some extensions
- several layout algorithm

Could you try to pick the main features and complete this list?

Reply:
I detailed the main features I know of, but I'm not sure I've included them all as I'm still discovering features those days! I've added a TODO, could you have a quick look @geofjamg to check? I've listed them below:

- creation of single line diagrams for given voltage levels, substations or zones in SVG format
- diagrams customization
     - several layout algorithms to generate the diagrams
     - several layout parameters to adjust the rendering
     - use your own component library or modify the existing
- small demo app to display the diagrams (note that this app is mainly for debug purposes and therefore is not included in the release)
- extensions to force positions in the diagram